docs/kubernetes.md

```diff
@@ -56,3 +56,109 @@ Pour accéder à votre cluster :
 Plus de détails sur l'administration kube avec k9s sur [cet article](https://opensource.com/article/20/5/kubernetes-administration).
 
 Une sonde [sentry-kubernetes](https://github.com/getsentry/sentry-kubernetes) est installée sur le cluster et permet de remonter toutes les erreurs : CronJob failed, probes... C'est une source d'information préciseuse quand quelque chose ne fonctionne pas dans vos déploiements. L'accès doit être demandé à la team SRE.
+
+## Variable d'environnement dans Kubernetes
+
+On vous recommande de récupérer vos variables d'environnement avec `envFrom`.
+
+```yaml
+# [...]
+  envFrom:
+    - configMapRef:
+        name: app-env
+    - secretRef:
+        name: app-env
+```
+
+### Variables de configuration
+
+Les variables qui configurent le projet dans l'environnement déployé. Ces variables sont prédictibles et non-chiffrées. Example : `NODE_ENV=PRODUCTION`
+
+Il est recommandé d'utiliser une [_ConfigMap_ par ensemble des variables du context](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#configure-all-key-value-pairs-in-a-configmap-as-container-environment-variables) et par environnement.
+
+```yaml
+# .k8s/environements/dev/app-env.configmap.yaml
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: app-env
+data:
+  NODE_ENV: "production"
+  GRAPHQL_ENDPOINT: "http://hasura/v1/graphql"
+  ACCOUNT_MAIL_SENDER: "contact@fabrique.social.gouv.fr"
+  FRONTEND_PORT: "${PORT}"
+  PRODUCTION: "false"
+```
+
+### Variables secrets
```

Comment:
```suggestion
### Sealed-secrets : Variables secretes
```

Reply:
Done in https://github.com/SocialGouv/support/commit/ce9385675ad930ae95632ee7dc15dc2b9685e377